anyway/flask_app.py

```diff
@@ -1236,9 +1303,27 @@ def oauth_callback(provider: str) -> Response:
 """
     Returns infographics-data API
 """
+parser = reqparse.RequestParser()
+parser.add_argument('id', type=int, help='News flash id')
+parser.add_argument('years_ago', type=int, default=5, help='Number of years back to consider accidents')
+
+resp_mod = api.model('infographics_data', {
+    'rank': fields.Raw(readonly=True, description='The task unique identifier'),
+    'task': fields.Raw(required=True, description='The task details')
+})
```

Comment:
Is this code describe the JSON returning from "/api/infographics-data" API? - I think some of the fields are missing

Reply:
Not sure I understand this part of resp_mod - should it represent infographics_data? What is task?

Reply:
This was an initial trial I forgot to remove. Removing it until I will document this API. I leave only documentation of input parameters for `infographics-data` API.